docs/observability/reliability-management-slo/index.md

```diff
@@ -129,6 +129,11 @@ Throughput is the amount of processing time by a service or system. Depending on
 Availability indicates if a service is working and handling valid requests. Other systems, services, and even virtual storage all have potential metrics to track with SLIs. The other option gives you the ability to include different SLOs based on your specific business needs.
 
 
+## Data Retention
+
+SLO data is retained for 800 days. It is retained separately, regardless of underlying logs/metrics retention.
```

Comment:
how about we also add  something like this to clarify which data are we talking about exactly

You can access this data via [SLO dashboards](https://help.sumologic.com/docs/observability/reliability-management-slo/dashboards/) or in [logs](https://help.sumologic.com/docs/observability/reliability-management-slo/create-slo/#slo-as-log-messages).

Reply:
thank you!